<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewWdgt</class>
 <widget class="QWidget" name="ReviewWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vertical_layout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="question_box">
       <item>
        <widget class="QLabel" name="question_label">
         <property name="text">
          <string>Question:</string>
         </property>
          <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWebEngineView2" name="question" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>50</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="answer_box">
       <item>
        <widget class="QLabel" name="answer_label">
         <property name="text">
          <string>Answer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWebEngineView2" name="answer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>295</width>
           <height>50</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="show_button">
       <property name="text">
        <string>Show &amp;answer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grades">
       <property name="title">
        <string>Grade your answer:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QVBoxLayout" name="_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton2" name="grade_0_button" native="true">
            <property name="text" stdset="0">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton2" name="grade_1_button" native="true">
            <property name="text" stdset="0">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton2" name="grade_2_button" native="true">
            <property name="text" stdset="0">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton2" name="grade_3_button" native="true">
            <property name="text" stdset="0">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton2" name="grade_4_button" native="true">
            <property name="text" stdset="0">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton2" name="grade_5_button" native="true">
            <property name="text" stdset="0">
             <string>5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView2</class>
   <extends>QWidget</extends>
   <header>mnemosyne/pyqt_ui/qwebengineview2</header>
  </customwidget>
  <customwidget>
   <class>QPushButton2</class>
   <extends>QPushButton</extends>
   <header>mnemosyne/pyqt_ui/qpushbutton2</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>show_button</sender>
   <signal>clicked()</signal>
   <receiver>ReviewWdgt</receiver>
   <slot>show_answer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
